.changeset/popular-cats-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sveltejs/adapter-cloudflare-workers': minor
+'@sveltejs/adapter-cloudflare': minor
+---
+
+feat: You can now pass the `external` property to the esbuild stage of the cloudflare adapters.  This allows you to mark a file or a package as external to exclude it from the build.  A typical use-case for this is the `fs` package that some modules import and which causes build-errors, even if you don't use that code-path.
```

Comment:
maybe we can short this up. we typically have a single sentence for changelog messages
```suggestion
feat: adds an `external` option to specify packages that should not be bundled as part of the build such as those provided by Node.js
```

Reply:
applied.